unittests/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ add_custom_command(
 file(GLOB UNIT_TESTS "*.cpp") # find all unit test suites
 add_executable( unit_test ${UNIT_TESTS} protocol_feature_digest_tests.cpp) # build unit tests as one executable
 
-target_link_libraries( unit_test eosio_chain_wrap chainbase eosio_testing fc appbase state_history abieos ${PLATFORM_SPECIFIC_LIBS} )
+target_link_libraries( unit_test eosio_chain_wrap chainbase eosio_testing fc appbase state_history producer_plugin abieos ${PLATFORM_SPECIFIC_LIBS} )
```

Comment:
For the unit tests, we don't really want them in the set of core unit tests for eos.  If you take a look at `trace_api_plugin` or `resource_monitor_plugin` for reference on what the file structure and CMake, etc. look like.

But, basically:
In producer_plugin directory, create a folder called `test` then add_subdirectory that in the main CMakeLists there.
You can then copy the structure of one of the test cpp files there and place your unit tests below there.

Reply:
Moved the unit test from core unit tests to a new `test` directory in `producer_plugin`.